scripts/bootstrap

```diff
@@ -8,3 +8,27 @@ docker-compose run --entrypoint=npm client install
 docker-compose run --entrypoint=bower client install --allow-root
 # Initialize database
 docker-compose run server rake db:setup
+# Boot app
+docker-compose start
+
+# Get docker-machine ip if there is one
+port="$(docker-compose port nginx 80 | cut -d: -f2)"
+[ -z "$DOCKER_MACHINE_NAME" ] && host='localhost' || host="$(docker-machine ip)"
```

Comment:
That doesn't work if you have several machines.
`Error: Expected to get one or more machine names as arguments`